Aptoide/appcoins-wallet-android
# [MOB-162] Added logic to handle asycn payment final step and redirect the flow to process the payment.

**What does this PR do?**

  Fixes the flow for the local payments in Brazil and Mexico:
 - By handling the the CodaPay redirect logic
 - Adding the back option in the first screen for payment with Boleto

**Database changed?**

No

**Where should the reviewer start?**

- [ ] WebViewActivity.kt
- [ ] BillingWebViewFragment.kt
- [ ] RemoteRepository.kt 

**How should this be manually tested?**

  

- With the use of a VPN connect to Brazil or Mexico and using production environment
- Make the processo of paying with Boletto and OXXO respectively
- Finish the flow by using CodaPay flow or pressing back
- See the async payment dialog

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-162](https://aptoide.atlassian.net/browse/MOB-162)


**Code Review Checklist**
